wagtail/admin/templates/wagtailadmin/userbar/base.html

```diff
@@ -22,13 +22,13 @@
                     <use href="#icon-wagtail-icon"></use>
                   </svg>
                 {% endblock %}
-                <span class="wagtail-userbar-help-text">{% trans 'Go to Wagtail admin interface' %}</span>
-            </div>
-            <div class='wagtail-userbar-items'>
+                <span class="wagtail-userbar-help-text">{% trans 'View Wagtail quick actions' %}</span>
```

Comment:
Just tried replacing this with the `aria-label` approach. Unfortunately, this approach causes my VoiceOver to only announce `menu pop-up collapsed, button` (without the label). Upon opening the menu it will announce the label. This is undesirable so I will try fixing the issue with the visually hidden text.

Reply:
Fixed in a43b33489f71af000faef13c9d71d7b567ff2bbc by updating the styles that visually hide the help text. Copied from https://css-tricks.com/inclusively-hidden/